.github/workflows/compose-packer-verify.yaml

```diff
@@ -0,0 +1,199 @@
+---
+name: Packer verify
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_call:
+    inputs:
+      GERRIT_BRANCH:
+        description: "Branch that change is against"
+        required: true
+        type: string
+      GERRIT_CHANGE_ID:
+        description: "The ID for the change"
+        required: true
+        type: string
+      GERRIT_CHANGE_NUMBER:
+        description: "The Gerrit number"
+        required: true
+        type: string
+      GERRIT_CHANGE_URL:
+        description: "URL to the change"
+        required: true
+        type: string
+      GERRIT_EVENT_TYPE:
+        description: "Type of Gerrit event"
+        required: true
+        type: string
+      GERRIT_PATCHSET_NUMBER:
+        description: "The patch number for the change"
+        required: true
+        type: string
+      GERRIT_PATCHSET_REVISION:
+        description: "The revision sha"
+        required: true
+        type: string
+      GERRIT_PROJECT:
+        description: "Project in Gerrit"
+        required: true
+        type: string
+      GERRIT_REFSPEC:
+        description: "Gerrit refspec of change"
+        required: true
+        type: string
+      comment-only:
+        description: "Make this workflow advisory only, default false"
+        required: false
+        type: string
+        default: "false"
+    secrets:
+      GERRIT_SSH_REQUIRED_PRIVKEY:
+        description: "SSH Key for the authorized user account"
+        required: true
+      CLOUDS_ENV_B64:
+        description: "Packer cloud environment credentials"
+        required: true
+      CLOUDS_YAML_B64:
+        description: "Openstack cloud environment credentials"
+        required: true
+
+env:
+  OS_CLOUD: "vex"
+  PACKER_VERSION: "1.9.1"
+
+concurrency:
+  # yamllint disable-line rule:line-length
+  # yamllint disable-line rule:line-length
+  group: compose-required-info-${{ github.workflow }}-${{ github.event.inputs.GERRIT_CHANGE_ID || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clear votes
+        # yamllint disable-line rule:line-length
+        uses: lfit/gerrit-review-action@6ac4c2322b68c0120a9b516eb0421491ee1b3fdf  # v0.4
+        with:
+          host: ${{ vars.GERRIT_SERVER }}
+          username: ${{ vars.GERRIT_SSH_REQUIRED_USER }}
+          key: ${{ secrets.GERRIT_SSH_REQUIRED_PRIVKEY }}
```

Comment:
This should be `GERRIT_SSH_PRIVKEY`

Reply:
this is no longer applicable since we have removed the voting step.